### PR TITLE
Feat: Completed Notes CRUD operation

### PR DESCRIPTION
**Pull request by vinc3nati**
Implemented: 
- Fetch Note
- Add Note
- Delete Note
- Update Note

image

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Clean and modular code 
Looks good to merge

File: src/App.js [hunk 1]
```diff
@@ -4,15 +4,30 @@ import "./App.css";
 import { Home } from "./pages/Home/Home";
 import { Footer } from "./components/Footer/Footer";
 import { Auth } from "./pages/Auth/Auth";
+import { useData } from "./contexts/data-context";
+import { Loader } from "./components/Loader/Loader";
+import { Notes } from "./pages/Notes/Notes";
```
Comment (PeeyushPrashant) [inline src/App.js:9]: for pages you can use barrel exports

**Review by PeeyushPrashant — APPROVED** [review]
variables naming is good
functions names are self explanatory
good to go

File: src/App.js (hunk 1 above)
Comment (vinc3nati) [reply src/App.js:9]: okay

**Pull request merged**